.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: java
 
 jdk:
-  - oraclejdk7
```

Comment:
:+1: 

Too bad Travis CI doesn't support Maven Toolchains out of the box (travis-ci/travis-ci#2727), we could have used a reactor build with 2 modules (or maybe even maven-invoker-plugin; note that this is possible by just shipping our own `toolchains.xml`).
I think we can also achieve this with an explicit call to `jdk_switcher use oraclejdk7` before running `mvn test` again in the `script` step of the `.travis.yml`.